images/php/config/laravel/template.apko.yaml

```diff
@@ -0,0 +1,46 @@
+contents:
+  packages:
+    - ca-certificates
+    - curl
+    - php-curl
+    - php-openssl
+    - php-iconv
+    - php-mbstring
+    - php-mysqlnd
+    - php-pdo
+    - php-pdo_sqlite
+    - php-pdo_mysql
+    - php-sodium
+    - php-phar
```

Comment:
```suggestion
```

Reply:
Related to the above change - I put these into `extra_packages` so that anything importing this can override the set without any conflicts.